### PR TITLE
Adding Filta. Contacts

### PR DESCRIPTION
**Pull request by padi**
Filta. is an outsourcing provider for eCommerce retailers and eCommerce agencies. They're interested in hosting meetups for developers even for those who do not use Shopify.

They're successfully organized the biggest Shopify Meetup in the world with 362 attendees. :D

### CODE REVIEW

**Comment by radamanthus** [comment]
@padi Thanks for this! 

2 small requests before merging:
- Use triple backticks for the address block, just like the Angat.io entry
- Add a "PhRUG contacts: @padi" section (You suggested this during our dinner :) )

**Comment by padi** [comment]
@radamanthus done!

File: organizing-meetups.md [hunk 1]
```diff
@@ -50,3 +50,25 @@ Freelancer
 ProudCloud
 
 SourcePad
+
+####  Filta
+PhRUG contacts: @padi
+
+##### Address
```
Comment (corroded) [inline organizing-meetups.md]: Just for consistency - maybe just make this a normal `p`? Or make the Angat one an h4 as well?

Comment (padi) [reply organizing-meetups.md]: Done!

**Pull request merged**